plugins/MidiImport/MidiImport.cpp

```diff
@@ -229,12 +229,12 @@ class smfMidiChannel
 			it = dynamic_cast<InstrumentTrack *>( Track::create( Track::Type::Instrument, tc ) );
 
 #ifdef LMMS_HAVE_FLUIDSYNTH
-			it_inst = it->loadInstrument( "sf2player" );
+			it_inst = it->loadInstrument( "soundFontplayer" );
```

Comment:
(Non-blocking) Per the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

```suggestion
			it_inst = it->loadInstrument("soundFontplayer");
```